Add giveawayDeleted event and edit embed when rerolling (#93)
Androz2091/discord-giveaways: README.md

* **options.embedColor**: a hexadecimal color for the embeds of giveaways.  
* **options.embedColorEnd**: a hexadecimal color the embeds of giveaways when they are ended.  
* **options.reaction**: the reaction that users will have to react to in order to participate.  

> * **options.requirements**: Extra data which you want to save regarding this giveaway. You can access it from the giveaway object of the giveaway created by giveaway.requirements.

Reviewer: ```suggestion
* **options.extraData**: Extra data which you want to save regarding this giveaway. You can access it from the giveaway object using `giveaway.extraData`.
```
